plugin/math-katex/math-katex.js

```diff
@@ -0,0 +1,523 @@
+/* global Reveal, katex */
+
+/**
+ * A plugin that renders mathematical formulas inside reveal.js slides using
+ * [KaTeX](https://github.com/Khan/KaTeX]).
+ *
+ * Calls `katex.renderToString` on DOM elements and regex matches and does some
+ * error handling.
+ *
+ * @author Johannes Schmitz
+ */
+window.RevealMath = window.RevealMath || (function() {
+	'use strict';
+
+	// --- Options and defaults ------------------------------------------------
+
+	var options = Reveal.getConfig().math || {};
+
+	// if (options.useCdn === true) {
```

Comment:
Oh thanks for the hint … I forgot to remove the comments …

This was refactored and the option is not needed anymore. The plugin automatically tries to load KaTeX locally and if that fails it makes a request to the CDN.